### ISSUE TITLE
New workspace wizard: can not Finish (button disabled)

### ISSUE THREAD

**Issue by frankgerhardt** [bndtools/bndtools#1385]
I'm trying to do the http://bndtools.org/tutorial.html and have created the api bundle. Eclipse shows an error:
```
   Missing Bnd Workspace. Create a new workspace with the 'New Bnd OSGi Workspace' wizard.	  
   bnd.bnd	/org.example.api	Unknown	Missing Bndtools Workspace Problem Marker
```
Then I try to create that workspace but I can't because I can not Finish the wizard. The Finish button is disabled. Only Cancel and Back are enabled.

![2016-05-26-184541_1366x768_scrot](https://cloud.githubusercontent.com/assets/550621/15583542/8ca58e72-2376-11e6-8e5e-20eb43392cae.png)

I'm lost. I can't continue with the tutorial. 

I'm confused. I have an Eclipse workspace already. Why do I need another workspace? Is that really a workspace??? There used to be a cnf project. Not any more?

**Comment by frankgerhardt** [comment]
Call me a hero, I restarted Eclipse and it works now.

**Comment by njbartlett** [comment]
1. The tutorial needs to be updated (sigh)
2. The legacy templates are broken... this will be fixed soon, possibly in 3.2.1
3. Yes the "Bnd Workspace" is basically just the cnf. It's not exactly the same as an Eclipse workspace.
